src/renderer/components/NodeDocumentation/NodeDocs.tsx

```diff
@@ -33,65 +34,75 @@ import { NodeFooter } from '../node/NodeFooter/NodeFooter';
 import { NodeHeader } from '../node/NodeHeader';
 import { TypeTag } from '../TypeTag';
 
-const customMarkdownTheme = {
-    p: (props: PropsWithChildren<unknown>) => {
-        const { children } = props;
+const SchemaLink = memo(({ schema }: { schema: NodeSchema }) => {
+    const { openNodeDocumentation } = useContext(NodeDocumentationContext);
+
+    return (
+        <Text
+            _hover={{
+                textDecoration: 'underline',
+            }}
+            as="i"
+            backgroundColor="var(--bg-700)"
+            borderRadius={4}
+            color="var(--link-color)"
+            cursor="pointer"
+            fontWeight="bold"
+            px={2}
+            py={1}
+            userSelect="text"
+            onClick={() => openNodeDocumentation(schema.schemaId)}
+        >
+            {schema.name}
+        </Text>
+    );
+});
+
+const customMarkdownTheme: Components = {
+    p: ({ children }) => {
         return (
             <Text
-                mb={0}
+                my={2}
                 userSelect="text"
             >
                 {children}
             </Text>
         );
     },
-    // If I don't give this a type of unknown, ReactMarkdown yells at me
-    // Just trust me, this works
-    a: memo((props: PropsWithChildren<unknown>) => {
-        const { children, href } = props as PropsWithChildren<{ href: string }>;
-
-        const { schemata } = useContext(BackendContext);
-        const { openNodeDocumentation } = useContext(NodeDocumentationContext);
-
-        const isInternalNodeLink = href.startsWith('#');
-        const linkedSchemaId = href.substring(1);
-
-        if (isInternalNodeLink) {
-            const nodeSchema = schemata.get(linkedSchemaId as SchemaId);
-            if (nodeSchema.schemaId !== '') {
-                return (
-                    <Text
-                        _hover={{
-                            textDecoration: 'underline',
-                        }}
-                        as="i"
-                        backgroundColor="var(--bg-700)"
-                        borderRadius={4}
-                        color="var(--link-color)"
-                        cursor="pointer"
-                        fontWeight="bold"
-                        px={2}
-                        py={1}
-                        userSelect="text"
-                        onClick={() => openNodeDocumentation(nodeSchema.schemaId)}
-                    >
-                        {nodeSchema.name}
-                    </Text>
-                );
-            }
-        }
-
+    // eslint-disable-next-line react/prop-types
+    a: memo(({ children, href }) => {
```

Comment:
I don't think we need to put the link thing here, it was just for the node links which have moved to code. The default should work fine.